core/jobs/src/main/java/com/cognifide/aet/job/common/collectors/accessibility/AccessibilityCollector.java

```diff
@@ -64,7 +63,8 @@ public CollectorStepResult collect() throws ProcessingException {
         .getExecutionResultAsString();
     final String json = jsExecutor.execute(script, standard).getExecutionResultAsString();
     List<AccessibilityIssue> issues = parseIssues(json);
-    getElementsPositions(issues, html);
+    AccessibilityIssueMarkupFinder issuesFinder = new AccessibilityIssueMarkupFinder(html,issues);
```

Comment:
How about moving `parseIssues(json)` logic to `AccessibilityIssueMarkupFinder`.

Then code here would look like:
```java
  List<AccessibilityIssue> issuesWithPositions = new AccessibilityIssueMarkupFinder(html, json).execute(); // or .find()/.get()/etc.
```